drivers/interrupt_controller/Kconfig.scr_ipic

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2023 Syntacore
+# SPDX-License-Identifier: Apache-2.0
+
+config SCR_IPIC
+	bool "Syntacore Integrated Programmable Interrupt Controller (IPIC)"
+	default y
+	depends on DT_HAS_SCR_IPIC_ENABLED
+	select MULTI_LEVEL_INTERRUPTS
+	select 2ND_LEVEL_INTERRUPTS
+	help
+	  Integrated Programmable Interrupt Controller provides support
+	  for external interrupt lines defined by the Syntacore specification.
+
+config IPIC_STATIC_LINE_MAPPING
+	bool
+	default y
+
+config EXT_IPIC_IRQ_LN_NUM
+	int
+	default 0
```

Comment:
Is there a reason to have these as hidden options instead of user selectable?

Reply:
This options initially were defined in platform config, but due to CI checks (and, of course, I agree with it) I put it here. When platform introduced, this options will be re-defined. I think, for example, for `2ND_LVL_ISR_TBL_OFFSET` mechanism is pretty the same. Can you please explain, what exactly do you mean?

Reply:
The problem is that your driver (at least in theory) is not exclusively linked to your platform, maybe tomorrow another (different) platform will want to use the same driver, so at least add an help here to explain what these options are doing and how to use those.